docs/sql-ref-syntax-aux-describe-table.md

```diff
@@ -60,8 +61,66 @@ to return the metadata pertaining to a partition or column respectively.
     and `col_name` are  mutually exclusive and can not be specified together. Currently
     nested columns are not allowed to be specified.
     
+    JSON format is not currently supported for individual columns.
+    
     **Syntax:** `[ database_name. ] [ table_name. ] column_name`
 
+* **AS JSON**
+
+  An optional parameter to return the table metadata in JSON format. Only supported when EXTENDED
+  format is specified.
+
+  **Syntax:** `[ AS JSON ]`
+
+  **Schema:**
+  
+  Notes:
+  * All string fields will be contained in double-quotes "...".
```

Comment:
This is JSON standard, we don't need to mention it. What we do need to mention is whether the identifier string (table name, catalog name, etc. )follows the SQL standard or not. e.g. `123abc` is not a valid SQL identifier and should be quoted as ``` `123abc` ```

Reply:
I think our implementation is `QuotingUtils.quoteIfNeeded`?